DigitalLearningSolutions.Data/Models/SectionContent/SectionContent.cs

```diff
@@ -6,32 +6,30 @@ public class SectionContent
     {
         public string CourseTitle { get; }
         public string SectionName { get; }
-        public int TimeMins { get; }
+        public int SectionTime { get; set; }
         public int AverageSectionTime { get; }
         public bool HasLearning { get; }
-        public double PercentComplete { get; }
-        public int DiagAttempts { get; }
-        public int SecScore { get; }
-        public int SecOutOf { get; }
-        public string DiagAssessPath { get; }
-        public string PLAssessPath { get; }
-        public int AttemptsPL { get; }
-        public int PLPassed { get; }
-        public bool DiagStatus { get; }
+        public int DiagnosticAttempts { get; set; }
+        public int SectionScore { get; set; }
+        public int MaxSectionScore { get; set; }
+        public string DiagnosticAssessmentPath { get; }
+        public string PostLearningAssessmentPath { get; }
+        public int PostLearningAttempts { get; }
+        public int PostLearningPassed { get; }
```

Comment:
For some reason the type was coming out as an int rather than a boolean. I think this might have something to do with weird SELECT statement that was being used which has to CAST to an int to use MAX. Dapper wasn't accepting a bool in the constructor, only an int

Reply:
Solution suggested in my other comment (on the SQL query) 🙂